tests/ignite/contrib/metrics/test_gpu_info.py

```diff
@@ -27,10 +27,11 @@ def no_site_packages():
 
 
 @pytest.mark.skipif(python_below_36, reason="No pynvml for python < 3.6")
-def test_no_pynvml_package(no_site_packages):
```

Comment:
As we are here, we can also remove `python_below_36` as we are now supporting >= 3.6 and droped 3.5